app/src/main/kotlin/com/wire/android/ui/userprofile/avatarpicker/AvatarPicker.kt

```diff
@@ -82,10 +82,10 @@ fun AvatarPickerScreen(viewModel: AvatarPickerViewModel) {
 // that I am not aware of ?
 fun onNewAvatarPicked(originalUri: Uri, targetAvatarPath: Path, scope: CoroutineScope, context: Context, viewModel: AvatarPickerViewModel) {
     scope.launch {
+        sanitizeAvatarImage(originalUri, targetAvatarPath, context)
```

Comment:
the issue was not in the order of the execution but in the case that changing the state was launched on a seperate coroutine but the order change slipped into my fix, so that is on me :P